src/transformers/models/ibert/utils/quant_modules.py

```diff
@@ -0,0 +1,538 @@
+import torch
```

Comment:
If this file was taken from fairseq, we'll need correct attribution in the copyright